budgetApp1/budgetApp1/CoreDataHandler.swift

```diff
@@ -46,26 +45,17 @@ class CoreDataHandler: CoreDataHandlerType {
         }
     }
     
-    func fetchAllUsers() -> Result<[UserEntity], Error> {
-        let fetchRequest: NSFetchRequest<UserEntity> = UserEntity.fetchRequest()
-        do {
-            let users = try context.fetch(fetchRequest)
-            return .success(users)
-        } catch {
-            return .failure(AuthError.failedTofetchUsers)
-        }
-    }
-    
-    func createUser(fullName: String, password: String, emailAddress: String, username: String) {
+    func createUser(fullName: String, password: String, emailAddress: String, username: String) -> Bool {
         if !checkIfUserHasAccount(username: username, password: password) {
             let user = UserEntity(context: context)
             user.username = username
             user.password = password
             user.fullName = fullName
             user.emailAddress = emailAddress
             saveContext()
+            return true
         } else {
```

Comment:
Since there is a return in the if, please remove the else and just return false

Reply:
Refactored accordingly.